### PR TITLE
Test changes for 'Fix error when trying to drop columns from temp tables.'

### PR DESCRIPTION
**Pull request by Sairakan**
### Description

Tests for https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/pull/410

### Issues Resolved

BABEL-4912

### Test Scenarios Covered ###
* **Use case based -**

* **Boundary conditions -**

* **Arbitrary inputs -**

* **Negative test cases -**

* **Minor version upgrade tests -**

* **Major version upgrade tests -**

* **Performance tests -**

* **Tooling impact -**

* **Client tests -**

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: test/JDBC/expected/temp_table.out [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-- BABEL-4912 test ALTER TABLE DROP COLUMN
+CREATE TABLE #t1 (a INT, b INT)
+GO
+
+ALTER TABLE #t1 DROP COLUMN b
```
Comment (timchang514) [inline test/JDBC/expected/temp_table.out:18]: can we add more tests, such as

- dropping then recreating the same column multiple times 
- make sure that dependencies aren't broken (so if the column depends on another object, or an object depends on the column that the proper behavior happens)
- more complicated dependencies such as with toast/index/identity etc
- ensure that the table also works (i can select, update, drop from it normally)

Comment (Sairakan) [reply test/JDBC/expected/temp_table.out:18]: Added more test cases. Note that foreign key constraints are not allowed for temp tables, so I can't test that.

File: test/JDBC/expected/temp_table_rollback-vu-verify.out [hunk 2]
```diff
@@ -153,37 +153,21 @@ BEGIN TRAN
 ALTER TABLE #temp_table_rollback_t1 DROP COLUMN b
 ROLLBACK
 GO
-~~ERROR (Code: 3726)~~
-
-~~ERROR (Message: cannot drop column b of table "#temp_table_rollback_t1" because other objects depend on it)~~
-
```
Comment (lejaokri) [inline test/JDBC/expected/temp_table_rollback-vu-verify.out:159]: thanks for handling this.
Do we have tests for scenarios where it's supposed to fail with this error?

Comment (Sairakan) [reply test/JDBC/expected/temp_table_rollback-vu-verify.out:159]: I just added an extra test case, that while not reproducing this exact error, produces a similar error due to a dependency.

**Review by lejaokri — APPROVED** [review]

**Pull request merged**